templates/includes/_faq.html

```diff
@@ -31,4 +31,14 @@
             </div>
         </div>
     </div>
+        <div class="panel panel-default">
+        <div class="panel-heading">
+            <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion" href="#apply"><strong>Q: I'm following Django Girls tutorial and something is not working in my code: can you help me debug it?</strong></a>
+        </div>
+        <div id="apply" class="panel-collapse collapse">
+            <div class="panel-body">
+                <p>A: Sadly, Django Girls Support Team won't be able to help you: our work is to help other organizers plan their workshop. If you need assistance while doing the tutorial, go to <a href="https://gitter.im/DjangoGirls/tutorial/">Django Girls Gitter room</a>: a group of volunteers is waiting for you there and will be happy to answer all your questions ;)</p>
+            </div>
+        </div>
+    </div>
 </div>
```

Comment:
I know it's out of scope of the PR, but I think this file needs a new line at the end! :)